src/reactors/ExclusiveDutchOrderReactor.sol

```diff
@@ -68,10 +68,6 @@ contract ExclusiveDutchOrderReactor is BaseReactor {
             revert DeadlineBeforeEndTime();
         }
 
-        if (order.decayEndTime < order.decayStartTime) {
```

Comment:
note removing this because it's duplicated in DutchDecayLib https://github.com/Uniswap/UniswapX/blob/3aa296eaef6b3634d4da996802e590e1aabd31d2/src/lib/DutchDecayLib.sol#L31